Formula/traefik.rb

```diff
@@ -0,0 +1,61 @@
+require "language/go"
+
+class Traefik < Formula
+  desc "Træfik, a modern reverse proxy"
+  homepage "https://traefik.io/"
+  url "https://github.com/containous/traefik/releases/download/v1.3.8/traefik-v1.3.8.src.tar.gz"
+  sha256 "6fce36dd30bb5ae5f91e69f2950f22fe7a74b920e80c6b441a0721122f6a6174"
+  head "https://github.com/containous/traefik.git"
+
+  depends_on "go" => :build
+  depends_on "node" => :build
+  depends_on "yarn" => :build
+
+  go_resource "github.com/jteeuwen/go-bindata" do
+    url "https://github.com/jteeuwen/go-bindata.git",
+        :revision => "a0ff2567cfb70903282db057e799fd826784d41d"
+  end
+
+  def install
+    ENV["GOPATH"] = buildpath
+    dir = buildpath/"src/github.com/containous/traefik"
+    dir.install buildpath.children
+    ENV.prepend_create_path "PATH", buildpath/"bin"
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    cd "src/github.com/jteeuwen/go-bindata/go-bindata" do
+      system "go", "install"
+    end
+
+    cd dir do
+      cd "webui" do
+        system "yarn", "install"
+        system "yarn", "run", "build"
+      end
+      system "go", "generate"
+      system "go", "build", "-o", bin/"traefik", "./cmd/traefik"
+      prefix.install_metafiles
+    end
+  end
+
+  test do
+    (testpath/"traefik.toml").write <<-EOF.undent
```

Comment:
should use `<<~EOS` not `<<-EOF.undent`

Reply:
Ok.
( I just did what other formulas have: there are "<<-EOF.undent", but no "<<~EOS" )

Reply:
Check the formulas now. Nothing using `.undent` anymore ;)

Reply:
Ok, at the time of pull request creation...